test/Meshes/interval.jl

```diff
@@ -126,6 +136,47 @@ end
     end
 end
 
+@testset "IntervalMesh ExponentialStretching reverse" begin
+    @test_throws ArgumentError unit_intervalmesh(
+        stretching = Meshes.ExponentialStretching(0.25),
+        nelems = 0,
+        reverse_mode = true,
+    )
+    mesh = unit_intervalmesh(
+        stretching = Meshes.ExponentialStretching(0.25),
+        nelems = 1,
+        reverse_mode = true,
+    )
+    @test Meshes.coordinates(mesh, 1, 1) == Geometry.ZPoint(-1.0)
+    @test Meshes.coordinates(mesh, 1, 2) ≈ Geometry.ZPoint(0.0)
+    # approx equal to gcm ref profile height fraction given the vert domain
+    # ~7.5 km of 45 km vertical domain extent
+    # we normalize to unit length for easy comparison
+    mesh = unit_intervalmesh(
+        stretching = Meshes.ExponentialStretching(7.5 / 45.0),
```

Comment:
Rather than hard coding a mesh for testing, it's better to test that the grid spacing is equidistant in exp(-z/H), where H is the scale height.

Reply:
Oh, I followed the same test pattern that was already there for the non-reversed stretched grid. I am not sure I entirely follow the suggestion though. Can you elaborate a bit more? We have the separate tests for the `ExponentialStretch` which scales with the height scale H

Reply:
Oh, I think I got it now! Basically check that the inverse transformation gives ewuispaced points.